src/qvi/qvi.ts

```diff
@@ -61,4 +110,49 @@ export class QVI {
                 _private
             );
     }
+
+    public async issueEngagementContextRoleCredential(
+        client: SignifyClient,
+        alias: string,
+        recipient: string,
+        credentialData:ECRVLEICredential,
+        source: ECR_AUTH_EDGES| ECR_LE_EDGES, //TODO: I believe the only input requiresd should be either the said of the auth credential or the said of the vlei credential. NOTE:I am not sure it makes sense to add the ECR_LE_EDGE since the LE would issue credentials to themselves 
```

Comment:
This might be better split into two properties.

In a case where an LE is issuing its own, the AUTH edge is not required, but the LE edge is always required, regardless of who (QVI/LE) is issuing the ECR.

I think it would also make a more descriptive API.

Reply:
this is a qvi sdk, why would an LE used this to issue themselves a credential?

Reply:
oooooof, really good question.